Jellyfin.Api/Controllers/DynamicHlsController.cs

```diff
@@ -629,7 +636,8 @@ [FromQuery] [RegularExpression(EncodingHelper.ValidationRegex)] string? subtitle
         [FromQuery] EncodingContext? context,
         [FromQuery] Dictionary<string, string> streamOptions,
         [FromQuery] bool enableAdaptiveBitrateStreaming = true,
-        [FromQuery] bool enableAudioVbrEncoding = true)
+        [FromQuery] bool enableAudioVbrEncoding = true,
+        [FromQuery] bool alwaysBurnInSubtitleWhenTranscoding = false)
```

Comment:
Do we need `alwaysBurnInSubtitleWhenTranscoding` in the audio enpoints?

Reply:
Probably not, but there are also other video options in audio endpoints